srcpkgs/v2ray/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'v2ray'
+pkgname=v2ray
+version=4.21.2
+revision=1
+build_style=fetch
+hostmakedepends="unzip"
+short_desc="Platform for building proxies to bypass network restrictions"
+maintainer="linarcx <linarcx@gmail.com>"
+license="MIT"
+homepage="https://github.com/v2ray/v2ray-core/"
+distfiles="${homepage}/releases/download/v${version}/${pkgname}-linux-64.zip"
```

Comment:
Please build from source (with `build_style=go`), this zip contains prebuilt linux 64-bit binaries.